app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,51 @@
+<h2><%= "#{@person.name}'s Tasks:" %></h2>
+
+<%# loop through all tasks and display task titles %>
+<div class="list">
+  <% @tasks.each do |task| %>
+    <ul class="tile <%= "complete" if task.complete == true %>">
+      <li>
+        <%= link_to task.title, task_path(task.id), class: "single-task" %>
+      </li>
+
+        <ul>
+
+          <% if task.complete != true %>
+            <%# if the task is not complete, we give the option to mark complete %>
+            <%= form_for task do |f| %>
+            <%# i have no freaking idea how this works, but I wanted to use the update action to mark the task completed instead of making a new action and this got the information I wanted to update into my params hash, so yay! %>
+
+                <%= f.hidden_field :completed_at, value: DateTime.now %>
+                <%= f.hidden_field :complete, value: true %>
+
+              <li>
+                <%= f.submit "COMPLETE", data: { confirm: "Good job! Task completed :)." } %>
+              </li>
+
+            <% end %>
+          <% else %>
+            <%# task is complete and so we give the user the option to mark incomplete %>
+            <%= form_for task do |f| %>
+
+                <%= f.hidden_field :completed_at, value: nil %>
```

Comment:
I learned something from your code here as well! I didn't know you could pass in an updated value like you did in this example, so that the completed_at value would get reset in the database.